src/components/RequestDescription.js

```diff
@@ -0,0 +1,66 @@
+import React, {useEffect, useRef, useState} from 'react';
+import {View} from 'react-native';
+import Form from './Form';
+import ScreenWrapper from './ScreenWrapper';
+import HeaderWithBackButton from './HeaderWithBackButton';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import TextInput from './TextInput';
+import styles from '../styles/styles';
+
+const propTypes = {
+    /* Onyx Props */
+    ...withLocalizePropTypes,
+};
+
+function RequestDescription(props) {
+    const inputRef = useRef(null);
+
+    // Same as NewtaskDescriptionPage, use the selection to place the cursor correctly if there is prior text
+    const [selection, setSelection] = useState({start: 0, end: 0});
+
+    // eslint-disable-next-line rulesdir/prefer-early-return
+    useEffect(() => {
+        if (props.task.report && props.task.report.description) {
+            const length = props.task.report.description.length;
+            setSelection({start: length, end: length});
+        }
+    }, [props.task.report]);
```

Comment:
`props.task` is not always defined, so this crashes the App when accessing the description page:

![Screenshot 2023-06-19 at 12 03 01 PM](https://github.com/Expensify/App/assets/22219519/dfd500bb-5c8b-4bc7-a09b-5b0baa41f723)

https://github.com/Expensify/App/assets/22219519/9753005b-92f7-4cb3-8042-a67e1dcaaf8d

Reply:
🤦